docker-compose-dev.yml

```diff
@@ -84,6 +84,8 @@ services:
       context: ../rems-smart-on-fhir
       dockerfile: Dockerfile.dev
     container_name: rems_dev_rems-smart-on-fhir
+    env_file:
```

Comment:
see my other comments but this shouldn't be necessary if everything defined in the .env file is already in the container. The .env defines the defaults which should be read in by the server anyways, making this declaration a bit redundant.